Client/Application/WebServer.swift

```diff
@@ -67,7 +67,7 @@ class WebServer {
     func registerMainBundleResourcesOfType(_ type: String, module: String) {
         for path: String in Bundle.paths(forResourcesOfType: type, inDirectory: Bundle.main.bundlePath) {
             let resource = NSURL(string: path)!.lastPathComponent
-            server.addGETHandler(forPath: "/\(module)/\(resource)", filePath: path as String, isAttachment: false, cacheAge: UInt.max, allowRangeRequests: true)
+            server.addGETHandler(forPath: "/\(module)/\(resource ?? "")", filePath: path as String, isAttachment: false, cacheAge: UInt.max, allowRangeRequests: true)
```

Comment:
@justindarc @sleroux  I want to suggest an alternative solution:

```
if let resource = NSURL(string: path)?.lastPathComponent {
    server.addGETHandler(forPath: "/\(module)/\(resource)", filePath: path as String,
        isAttachment: false, cacheAge: UInt.max, allowRangeRequests: true)
} else {
    ...log that we are unable to register path...
}
```

The original patch dereferences `resource` (an Optional) and if it is `nil` then it will use the empty string `""` as `resource`. But I don't think that makes sense. Because we are then registering a non handler with invalid path. There is no point to that.